go.mod

```diff
@@ -1,18 +1,18 @@
 module github.com/crc-org/machine-driver-libvirt
 
 require (
-	github.com/crc-org/machine v0.0.0-20221028075518-f9b43442196b
+	github.com/crc-org/machine v0.0.0-20240926103419-a943b47fd48b
 	github.com/sirupsen/logrus v1.9.3
-	github.com/stretchr/testify v1.8.4
-	libvirt.org/go/libvirt v1.8000.0
-	libvirt.org/go/libvirtxml v1.9008.0
+	github.com/stretchr/testify v1.9.0
+	libvirt.org/go/libvirt v1.11001.0
```

Comment:
careful with libvirt upgrades, we need a version that is available in rhel8 or 9

Reply:
I didn't update it because in RHEL-8 it is still libvirt version is 8.0